fee_allocator/accounting/core_pools.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import annotations
+from dataclasses import dataclass, field
+from web3 import Web3
+from typing import List, Dict, TYPE_CHECKING
+from decimal import Decimal
+from datetime import datetime
+
+from bal_tools.models import PoolSnapshot, TWAPResult
+from fee_allocator.accounting.interfaces import AbstractCorePool
+from fee_allocator.accounting.overrides import CorePoolOverride, overrides
+
+if TYPE_CHECKING:
+    from fee_allocator.accounting.chains import Chain
+
+
+@dataclass
+class CorePoolData:
+    pool_id: str
+    label: str
+    bpt_price: Decimal
+    tokens_price: List[TWAPResult]
+    gauge_address: str
+    start_snap: PoolSnapshot
+    end_snap: PoolSnapshot
+    last_join_exit_ts: int
+
+    address: str = field(init=False)
+    earned_bpt_fee: Decimal = field(init=False)
+    earned_bpt_fee_usd: Decimal = field(init=False)
+    earned_tokens_fee: Dict[str, Decimal] = field(init=False)
```

Comment:
Does this represent fees collected in native tokens?

Reply:
yes
`earned_tokens_fee`: token fees accrued by pool in native token
`earned_tokens_fee_usd`:  token fees accrued by pool in usd

Reply:
renamed in [267533b](https://github.com/BalancerMaxis/protocol_fee_allocator_v2/pull/1/commits/267533b4988923a03b0ed0038c246fb1efd36910)